start_miner.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# Load environment variables from .env file
+set -a
+source miner.env
+set +a
+
+# Check if the process is already running
+if pm2 list | grep -q "bitmind_miner"; then
+  echo "Process 'bitmind_miner' is already running. Deleting it..."
+  pm2 delete bitmind_miner
+fi
+
+# Start the process with arguments from environment variables
+pm2 start python --name bitmind_miner -- neurons/miner.py \
+  --neuron.model_path $MODEL_PATH \
+  --netuid $NETUID \
+  --subtensor.network $SUBTENSOR_NETWORK \
```

Comment:
Can we please add a flag for `--subtensor.chain_endpoint $SUBTENSOR_CHAIN_ENDPOINT \`?

A new env var would need to be added also